7-objects.js

```diff
@@ -0,0 +1,102 @@
+var Recipe = {
+    title: "Sunday Lunch",
+    servings: 4,
+    ingredients: "Meat, gravy, yorkshire puddings"
+};
+
+function Node (data) {
+    this.data = data;
+    this.next = null;
+    this.previous = null;
+};
+
+function SingleLinkedList () {
+    var me = this;
+    me.head = new Node("head");
+
+    me.find = function (item) {
+        var currentNode = me.head;
+        while (currentNode.data != item){
+            currentNode = currentNode.next;
+        }
+        return currentNode;
+    },
+
+    me.insert = function (newData, item) {
```

Comment:
This is normally the responsibility of the node object - it inserts stuff after it.

Can clearly see you know what you are doing, so no need to fix.